exporters/otlp/protocoldriver.go

```diff
@@ -51,6 +51,25 @@ type ProtocolDriver interface {
 	ExportTraces(ctx context.Context, ss []*tracesdk.SpanSnapshot) error
 }
 
+// Provides options for setting up a split driver
+type Option func(d *splitDriver)
```

Comment:
```suggestion
// Provides options for setting up a split driver
type Option func(d *splitDriver)
```

This should probably go in `options.go` and the type renamed to `ProtocolDriverOption` since there are multiple types of options in this package.